src/sentry/integrations/aws_lambda/utils.py

```diff
@@ -36,6 +44,26 @@ def get_arn_without_version(arn):
     return arn.rpartition(":")[0]
 
 
+def get_version_of_arn(arn):
+    return int(arn.split(":")[-1])
+
+
+def get_latest_layer_for_function(function):
+    region = parse_arn(function["FunctionArn"])["region"]
+    runtime = function["Runtime"]
+    if runtime.startswith("nodejs"):
+        return get_aws_node_arn(region)
+    # update when we can handle other runtimes like Python
+    raise Exception("Unsupported runtime")
+
+
+def get_latest_layer_version(runtime):
+    if runtime.startswith("nodejs"):
+        return int(options.get("aws-lambda.node-layer-version"))
```

Comment:
@scefali do we need to check node version is in `SUPPORTED_RUNTIMES`? Should `nodejs8` raise exception?

Reply:
@NisanthanNanthakumar Nah, we check the runtimes at a different point earlier on. All Node runtime should use the same layer regardless. This is here so that if/when we roll out this feature for Python, we only have to update the code in a small number of functions like this one.